metabase/metabase
# Nudge larger basic instances to Pro

Closes #31264

### Description

Insert a “nudge to pro” card on the admin page when managing more than 50 active users.

### How to verify

1. Go to http://localhost:3000/admin/people
2. Force the `shouldNudgeToPro` selector return true or false in code.
3. Ensure the card is present for both People and Groups page.

### Demo

**With nudge:**

<img width="1201" alt="CleanShot 2023-06-16 at 17 12 59@2x" src="https://github.com/metabase/metabase/assets/116838/59d9a31a-48de-4969-9730-ff3d2b4416f7">

**Without nudge:**

<img width="1317" alt="CleanShot 2023-06-16 at 17 14 51@2x" src="https://github.com/metabase/metabase/assets/116838/4b91b401-bdd1-4502-b378-b20885a41204">

### Checklist

- [x] Check what the correct `utm_media` value should be
